src/containers/shared/components/SearchResults/SearchResults.tsx

```diff
@@ -0,0 +1,291 @@
+import { useEffect, useRef, useState } from 'react'
+import { Link } from 'react-router-dom'
+import { convertHexToString } from 'xrpl'
+import Logo from '../../images/no_token_logo.svg'
+import './SearchResults.scss'
+
+import { Amount } from '../Amount'
+import { localizeNumber } from '../../utils'
+
+interface SearchResultsProps {
+  currentSearchValue: string
+  setCurrentSearchInput: (string) => void
+}
+
+interface SearchResultBarProps {
+  resultContent: any
+  onClick: () => void
+  xrpPrice: number
+}
+const SearchResultBar = ({
+  resultContent,
+  onClick,
+  xrpPrice,
+}: SearchResultBarProps): JSX.Element => {
+  const parsePrice = (price): number => {
+    const parsed = Number(price).toFixed(6)
+    if (Number(parsed) === 0) {
+      return 0
+    }
+    return Number(Number(parseFloat(parsed) * xrpPrice).toFixed(6))
+  }
+
+  const parseDomain = (domain: string): string => {
```

Comment:
There should already be existing parse domain logic

Reply:
I see a DomainLink component in the repo, but the requirement here was to remove the 'https' protocol prefix visually while that one has logic to add it in. Can add my logic in this component as a new prop (stripLinkProtocol), unless there was different domain parsing logic you meant?

Reply:
The domain on validators page has https stripped off so you can reuse the logic there:
https://livenet.xrpl.org/network/validators

Reply:
Actually nevermind, the data fetched from VHS already has domain stripped off

Reply:
I would add that logic to the component and have it optionally strip it.  The stripping can also be done via a regex expression vs checking each possible protocol.